src/webportal/src/app/vc/vc.component.scss

```diff
@@ -101,4 +101,4 @@ ul, li{
     color: #c0c0c0;
     cursor: default;
   }
-}
+}
```

Comment:
why the newline at end of file is removed?

Reply:
interesting, I will add it back